src/sentry/notifications/notificationcontroller.py

```diff
@@ -446,10 +446,13 @@ def get_subscriptions_status_for_projects(
                     any(
                         value == NotificationSettingsOptionEnum.ALWAYS for value in setting.values()
                     ),
+                    all(
+                        value == NotificationSettingsOptionEnum.NEVER for value in setting.values()
+                    ),
                 )
 
             if not has_setting:
-                subscription_status_for_projects[project] = (False, False)
+                subscription_status_for_projects[project] = (True, False, True)
```

Comment:
@snigdhas this tuple is hard to understand. Can we make it some object with named parameters instead?